tests/validators/tests.py

```diff
@@ -407,6 +410,8 @@
     (validate_ipv46_address, "fe80::1", None),
     (validate_ipv46_address, "::1", None),
     (validate_ipv46_address, "1:2:3:4:5:6:7:8", None),
+    (validate_ipv46_address, ipaddress.IPv6Address("::ffff:2.125.160.216"), None),
+    (validate_ipv46_address, ipaddress.IPv6Address("fe80::1"), None),
```

Comment:
Would you add a few cases for IPv4 instances?

Reply:
Added :+1:

Reply:
Thank you, marking the ticket as Ready for checkin.